docs/rfc/rfc-1.adoc

```diff
@@ -401,13 +412,151 @@ to submit multiple redemptions. After a redemption, if the wallet has under
 `wallet_min_btc` remaining, it transfers that BTC to a randomly selected wallet
 and closes.
 
+Each redemption request is identified by a concatenation of the wallet's pubkey
+hash and redeemer's output hash (redeemer's BTC address). Such an identifier
+allows retrieving pending redemption requests in a gas-efficient way based on
+information provided in the redemption proof. A consequence of this approach is
+that the redeemer can not use the same redemption BTC address when there is
+already one redemption request pending from the given wallet. User experience
+can be improved by the dApp by selecting the previous-oldest wallet without a
+pending redemption request to the given BTC address in case a new redemption is
+requested before the pending one gets cleared out.
+
+At a minimum, pending redemption request needs to capture the following
+information:
+
+- Expected range of value for redemption UTXO: needed to validate the amount
```

Comment:
Check for my understanding: Is this a range because the mining fee is a range? As in, say that we're trying to redeem 1 TBTC, the redemption fee is 1% (0.01 TBTC), and the mining fee ranges from 0.0001 BTC to 0.0003 BTC (paid in TBTC). 

We need to validate that the redemption address receives from `1-0.01-0.0001=0.9899` to `1-0.01-0.0001=0.9897` BTC.

Reply:
Yes, exactly as you wrote, it's because we don't know the mining fee. Piotr explained this to me in this comment:
https://github.com/keep-network/tbtc-v2/pull/132#discussion_r803873543

Reply:
Correct. In practice, we store the current redemption (treasury fee) and max BTC network fee in the `RedemptionRequest` but it is easier to explain in English as an acceptable range given that in the end, we are validating a range calculated based on the captured values of the parameters:

https://github.com/keep-network/tbtc-v2/blob/73fcd4832ad1a1cea43d9826b49f634453337f24/solidity/contracts/bridge/Bridge.sol#L113-L125

https://github.com/keep-network/tbtc-v2/blob/73fcd4832ad1a1cea43d9826b49f634453337f24/solidity/contracts/bridge/Bridge.sol#L1433-L1441

`txMaxFee` should be set to an exceptionally high value, say 1000x expected fee. It is there only to protect against wallets controlled by Bitcoin miners and stealing all the redeeming value in a form of a fee.

Reply:
One question though: should the redemption fee be a % of the redeemed value of a fixed fee in sat? Right now, the latter is implemented.

Reply:
In my head it was a percentage of redeemed value - that's how I did all of the profitability modeling (otherwise it incentivizes big redemptions and now it's based on number of redemptions rather than amount redeemed)

Reply:
We will have to update it @lukasz-zimnoch

Reply:
Here it is https://github.com/keep-network/tbtc-v2/pull/136